### PR TITLE
#170764564 Setup postgreSQL db engine

### PR DESCRIPTION
**Pull request by Elvis-rugamba**
#### What does this PR do?
Setup Postgresql Database as DB engine
#### Description of task to be completed

- Install pg dependency

- Install dotenv

- Create a .env and specify a database url

- Write a script file that connects the application with the database

#### How can this be manually tested?
After cloning repo, Cd into it,

- RUN `npm install`,

- Open .env and edit the DB_USER, DB_PASSWORD, DB_HOST, DB_PORT, DB_DATABASE to match your respective database information

- RUN `npm run db:connect`

#### What are the relevant Pivotal Tracker Stories?
#170764564
#### Screenshots (if appropriate)
None

### CODE REVIEW

**Review by shaazk — APPROVED** [review]
looks good to me

**Review by Skemc — APPROVED** [review]
Looks good bruv

File: package.json [hunk 1]
```diff
@@ -17,20 +18,22 @@
...
+    "cross-env": "^6.0.3",
     "eslint": "^6.8.0",
     "eslint-config-airbnb": "^18.0.1",
     "eslint-plugin-import": "^2.20.0",
+    "nodemon": "^1.19.4",
     "mocha": "^7.0.0",
-    "nyc": "^15.0.0",
-    "nodemon": "^1.19.4"
+    "nyc": "^15.0.0"
+    "coveralls": "^3.0.9"
```
Comment (ajimae) [inline package.json]: There's is an error in this file, please fix it.

Thanks

File: package.json [hunk 2]
```diff
@@ -6,6 +6,7 @@
   "dependencies": {    
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
+    "pg": "^7.17.1"
     "swagger-ui-express": "^4.1.3"
```
Comment (ajimae) [inline package.json]: There is an error here, please fix

**Review by ajimae — CHANGES_REQUESTED** [review]
Please fix all the errors in this PR.
Thank you.

File: package.json (hunk 1 above)
Comment (Elvis-rugamba) [reply package.json]: Resolved

File: package.json (hunk 2 above)
Comment (Elvis-rugamba) [reply package.json]: Resolved

File: package.json (hunk 1 above)
Comment (ajimae) [reply package.json]: It's not resolved, I can still see the error

**Review by ajimae — APPROVED** [review]
This looks good to me!

**Pull request merged**